browser/search/ntp_utils.cc

```diff
@@ -5,32 +5,96 @@
 
 #include "brave/browser/search/ntp_utils.h"
 
+#include "base/logging.h"
 #include "brave/browser/profiles/profile_util.h"
 #include "brave/common/pref_names.h"
+#include "brave/components/crypto_dot_com/browser/buildflags/buildflags.h"
 #include "chrome/common/pref_names.h"
 #include "components/pref_registry/pref_registry_syncable.h"
 #include "components/prefs/pref_registry_simple.h"
 #include "components/prefs/pref_service.h"
 
+#if BUILDFLAG(CRYPTO_DOT_COM_ENABLED)
+#include "brave/components/crypto_dot_com/common/pref_names.h"
+#endif
+
 namespace {
 
 void ClearNewTabPageProfilePrefs(Profile* profile) {
   PrefService* prefs = profile->GetPrefs();
   prefs->ClearPref(kNewTabPageShowTopSites);
 }
 
+const char* const widget_pref_names[] = {
```

Comment:
nit: `kWidgetPrefNames` is appropriate for global vars name convension.

Reply:
fixed